resources/js/tasks/components/TasksPreview.vue

```diff
@@ -6,15 +6,22 @@
       v-model="showPreview"
       :right="showRight"
       shadow
+      no-header
     >
       <template #default="{ hide }">
         <div class="p-3">
-          <div>
+          <div class="w-100 h-100 mb-3">
             <!-- Set Controls to navigate -->
-            <i
-              class="fa fa-close"
-              @click="hide"
-            />
+            <div class="text-right align-items-center">
+              <button
+                type="button"
+                aria-label="Close"
+                class="px-2 py-1 close"
+                @click="hide"
+              >
+                ×
```

Comment:
use icon

Reply:
The x icon does not show up, this is the button that appears to be used in many windows